### DISCUSSION TITLE
Redirect call to vim.notify with specific title to mini view

### DISCUSSION THREAD

**Discussion by LordMZTE** [folke/noice.nvim#481] [Q&A] [answered]
I'm using `nvim-dap` which has the habit of producing 3 huge notifications via `vim.notify` when performing a hot code reload. I would like to detect notifications which have their title set to "DAP" and redirect those to the less obstrusive `mini` view instead.

**Comment by folke** [comment]
```lua
{
  routes = {
    {
      filter = {
        event = "notify",
        find = "DAP",
      },
      view = "mini",
    },
  },
}
```

**Comment by LordMZTE** [comment]
This doesn't detect `DAP` in the title, only in the body. If I call `vim.notify("test", 2, { title = "DAP" })`, this still shows as a notification.

**Reply by folke** [reply, marked as answer]
Right, for the title, you can use:

```lua
    table.insert(opts.routes, {
      filter = {
        event = "notify",
        ---@param message NoiceMessage
        cond = function(message)
          return message.opts and message.opts.title == "DAP"
        end,
      },
      opts = { skip = true },
    })
```

This matches all notifications with title = "DAP"

**Reply by LordMZTE** [reply]
Works a charm! (using `view = "mini"` instead of skipping it)
